### ISSUE TITLE
Set export precision using arguments

### ISSUE THREAD

**Issue by BigRoy** [iimachines/Maya2glTF#192]
### Issue

It would be great if we could set the [different precisions defined for the export](https://github.com/iimachines/Maya2glTF/blob/406dd2704b5a1dde5f4449903ab79e42554cf34c/src/BasicTypes.h#L34-L39) using arguments to the export command.

So that would be exposing optional flags like this:
```
-posPrecision 1e9
-dirPrecision 1e9
-colPrecision 1e9
-texPrecision 1e9
-sclPrecision 1e9
-matPrecision 1e9
```
Which default to the currently hardcoded value if not set.

**Comment by ziriax** [comment]
Indeed, PR welcome 😁

**Comment by BigRoy** [comment]
Resolved with #194
